runtime/onert/backend/cpu/ops/OperationUtils.h

```diff
@@ -194,7 +194,39 @@ bool HaveSameShapes(const Tensor *input1, const Tensor *input2);
 
 int32_t CalculateInputRadius(int input_integer_bits, int input_left_shift);
 
-uint32_t sizeOfData(OperandType type, const std::vector<uint32_t> &dimensions);
```

Comment:
@ragmani Can't we fix this vector type to `vector<int32_t>` not using template?

Reply:
Yes, we can fix it to `vector<int32_t>` because this function is not being used now. Do you want it?

Reply:
Yes. I prefer to not using template.